spring-cloud-dataflow-tasklauncher/spring-cloud-dataflow-tasklauncher-sink/pom.xml

```diff
@@ -0,0 +1,352 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.cloud</groupId>
+		<artifactId>spring-cloud-dataflow-tasklauncher</artifactId>
+		<version>2.9.0-SNAPSHOT</version>
+	</parent>
+	<artifactId>spring-cloud-dataflow-tasklauncher-sink</artifactId>
+	<properties>
```

Comment:
Looking all these versions defined here, this smells a lot. Generally speaking in dataflow modules every dependency version should come from main dataflow parent pom or its parents or dep management. This is so that we don't hide things in child poms(as we always forget to change things there as generally we only look parents for versions).

`spring-cloud-dataflow-build` gives use boot and cloud depd and with some common deps. If build project doesn't give deps, then those should be defined in dataflow parent pom.